### PR TITLE
fix L1 bridge function input use

### PR DESCRIPTION
**Pull request by albertnbrown**
consistent use of _l1token var
made sure revert comments all give contract name

### CODE REVIEW

File: contracts/bridge/L2ECOBridge.sol [hunk 1]
```diff
@@ -227,16 +234,16 @@ contract L2ECOBridge is IL2ECOBridge, CrossDomainEnabledUpgradeable {
         bytes calldata _data
     ) internal {
         // Burn the withdrawn tokens from L2
-        l2EcoToken.burn(msg.sender, _amount);
+        l2Eco.burn(msg.sender, _amount);
 
         // Construct calldata for l1TokenBridge.finalizeERC20Withdrawal(_to, _amount)
-        address l1Token = l2EcoToken.l1Token();
+        address l1Token = l2Eco.l1Token();
```
Comment (StoyanD) [inline contracts/bridge/L2ECOBridge.sol:240]: was l1Token() supposed to remain on the l2Eco contract?

Comment (StoyanD) [reply contracts/bridge/L2ECOBridge.sol:240]: dont we already have this in l1Eco?

**Review by StoyanD — APPROVED** [review]
have a question on _initiateWithdrawal

also for L2ECOBridge:finalizeDeposit can u switch the order of the isL1EcoToken and isL2EcoToken modifiers to that they are called in numerical order?

```
    function finalizeDeposit(
        address _l1Token,
        address _l2Token,
        address _from,
        address _to,
        uint256 _amount,
        bytes calldata _data
    )
        external
        virtual
        onlyFromCrossDomainAccount(l1TokenBridge)
       isL1EcoToken(_l1Token)
        isL2EcoToken(_l2Token)
       
    {
```

File: contracts/bridge/L2ECOBridge.sol (hunk 1 above)
Comment (albertnbrown) [reply contracts/bridge/L2ECOBridge.sol:240]: good catch

**Review by StoyanD — APPROVED** [review]

**Pull request merged**